notifications_utils/polygons.py

```diff
@@ -25,8 +25,10 @@
         # These are the names of coordinate reference systems, which
```

Comment:
We don't mention "degrees" (or lat / long) anywhere. Ideally we should avoid people have to research WSG84 to find out what the units are in order to understand these comments.

Could we mention it somewhere? Even better, could we mention it in [lots](https://github.com/alphagov/notifications-utils/blob/943ceeef60327596990af32702ae89d6bf9891a5/notifications_utils/polygons.py#L346) [of](https://github.com/alphagov/notifications-utils/blob/943ceeef60327596990af32702ae89d6bf9891a5/notifications_utils/polygons.py#L161) [places](https://github.com/alphagov/notifications-utils/blob/943ceeef60327596990af32702ae89d6bf9891a5/notifications_utils/polygons.py#L113) so it's really easy for someone to read through this class and understand what's going on? Most of the methods for the class have comments to explain what we do, so it looks like there's a precedent.